presto-hive-hadoop2/bin/run_hive_with_azure_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail -x
+
+. ${BASH_SOURCE%/*}/common.sh
+
+cd ${PROJECT_ROOT}
+
+./mvnw -B -pl presto-hive-hadoop2 test -P test-hive-hadoop2-hdinsight \
```

Comment:
what's the difference between this script and `presto-hive-hadoop2/bin/run_hive_wasb_with_azure_tests.sh`?